sql/telemetry_derived/main_summary_v4/part1.sql

```diff
@@ -720,7 +720,7 @@ SELECT
     NULL AS room_share
   ) AS loop_activity_counter
 FROM
-  `moz-fx-data-shared-prod.telemetry_stable.main_v4`
+  `moz-fx-data-shared-prod.telemetry.main`
```

Comment:
Changing main_summary to hit the user-facing view rather than the stable table so that it gets the nulled sessionLengths.

Looking especially for @relud's okay on this one.